core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -290,6 +291,14 @@
    * Default value of the percentage operations accessing the hot set.
    */
   public static final String HOTSPOT_OPN_FRACTION_DEFAULT = "0.8";
+  
+  private Hashtable<String, String> _operations = new Hashtable<String, String>() {{
```

Comment:
AFAICT, this map does not need to be synchronized. Replace it with final Map and instantiate it in an initializer block where you can wrap it in Collections.unmodifiableMap